### PR TITLE
feat: property schema defined

### LINKED ISSUE #14: Schema/Model Properties

### PR DESCRIPTION
**Pull request by JasterV**
fixes #14

### CODE REVIEW

File: src/models/properties-model.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    enum: ["private", "co_working", "security_system"],
+    required: true,
+  },
+});
+
+const Property = mongoose.model("properties", PropertySchema);
+
+const Home = Property.discriminator("Home", HomeSchema);
+
+const Office = Property.discriminator("Office", OfficeSchema);
```
Comment (JasterV) [inline src/models/properties-model.js:165]: Good job dude! :rocket:

**Review by JasterV — COMMENTED** [review]
Looks good :D

File: src/models/properties-model.js (hunk 1 above)
Comment (Damantino) [reply src/models/properties-model.js:165]: Thanks!

**Review by devmxhv — APPROVED** [review]
Good Job!

**Review by devmxhv — APPROVED** [review]
Approved Changes

**Review by devmxhv — APPROVED** [review]
Approved Changes

**Pull request merged**